test/libsolidity/smtCheckerTests/control_flow/return_1.sol

```diff
@@ -0,0 +1,22 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function add(uint x, uint y) internal pure returns (uint) {
+		if (y == 0)
+			return x;
+		if (y == 1)
+			return ++x;
+		if (y == 2)
+			return x + 2;
+		return x + y;
+	}
+
+	function f() public pure {
+		assert(add(100, 0) == 100);
+		assert(add(100, 1) == 101);
+		assert(add(100, 2) == 102);
+		assert(add(100, 100) == 200);
+	}
+}
+// ----
+// Warning 2661: (158-161): BMC: Overflow (resulting value larger than 2**256 - 1) happens here.
```

Comment:
Have we created an issue for the fact that CHC engine does not creates overflow targets for `++` expressions?

Reply:
https://github.com/ethereum/solidity/issues/10514 just did